src/vtools-analysis.py

```diff
@@ -169,6 +170,16 @@ def summarize(infile, df, config_dict, debug):
     vals.append(frame_drop_average_length)
     keys.append("frame_drop_text_list")
     vals.append(frame_drop_text_list)
+    if config_dict['dump_audio_info'] == True:
+        sample_rate, bitrate, duration = (
+            vtools_ffprobe.get_audio_info(infile)
+        )
+        keys.append("audio_sample_rate")
+        vals.append(sample_rate)
+        keys.append("audio_bitrate")
+        vals.append(bitrate)
+        keys.append("audio_duration")
```

Comment:
audio_duration_time for coherence with above

Reply:
sure, will update